csvqb/csvqb/.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
```

Comment:
I generally think it's a good idea to keep the user's IDE configuration files out of source control, else developers are always fighting over what their settings are and you end up with loads of conflicts. Perhaps we should add `.vscode` to `.gitignore`?